docs/standard/data/sqlite/bulk-insert.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Bulk Insert
+title: Bulk insert
```

Comment:
this is a very short article @bricelam. I'd consider moving this into another article.

Reply:
A lot of people ask how to do bulk insert. This is just a page for them to find saying you don’t have to do anything special on SQLite.

I think if we add a bit more context of what bulk insert is on other databases, how it speeds things up and why it doesn’t apply to SQLite it would be a better article. Also maybe it could use some more details on how these two steps help speed things up.

Reply:
But maybe it can also merge into the ADO.NET limitations